module_utils/vcd.py

```diff
@@ -26,6 +27,7 @@ def __init__(self, *args, **kwargs):
         argument_spec = vcd_argument_spec()
         argument_spec.update(kwargs.get('argument_spec', dict()))
         kwargs['argument_spec'] = argument_spec
+        urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```

Comment:
could you please call it in global space?

Reply:
Why global?
All modules inherit `VcdAnsibleModule` anyway, as it handles common module spec & login, so init will be called as soon as they're created.

Reply:
correct but it is a module `urllib3` level setting and I think it would be better if we set it outside the class.

Reply:
done, squashed commits